src/nodes/hypertable_modify.c

```diff
@@ -401,6 +422,35 @@ ts_hypertable_modify_fixup_tlist(Plan *plan)
 	}
 }
 
+/* ROWID_VAR only exists in PG14+ */
+#if PG14_GE
+List *
+ts_replace_rowid_vars(PlannerInfo *root, List *tlist, int varno)
```

Comment:
```suggestion
ts_replace_rowid_vars(PlannerInfo *root, List *tlist, Index rt_index)
```
I don't really understand the arguments of this function. Maybe rename them some way? Or add some comment?

Reply:
Those are very common names in the planner PlannerInfo is almost always named root and targetlists are commonly abbreviated as tlist.